### PR TITLE
Add empty contoller instead of sidebar

### PR DESCRIPTION
**Pull request by akaDuality**
Screen Shot 2022-08-11 at 01 49 16

- Create StateViewController that allows to simplify state changing
- Just create simple EmptyViewController

Possible states of right side bar can be extended in feature

### CODE REVIEW

**Comment by akaDuality** [comment]
@bbsody How do you think, is it ok to show placeholder text? Or we should disabled state of every view in it?

File: VoiceOver Designer/Features/Sources/Settings/StateViewController.swift [hunk 1]
```diff
@@ -0,0 +1,87 @@
...
+    private func addNew(_ newController: ViewController) {
+        addChild(newController)
+        view.addSubview(newController.view)
+        view.pinToBounds(newController.view)
+//        newController.didMove(toParent: self)
+        
+        currentController = newController
+    }
+    
+    private func removeCurrenIfExists() {
```
Comment (drucelweisse) [inline VoiceOver Designer/Features/Sources/Settings/StateViewController.swift:56]: typo in function name

File: VoiceOver Designer/Features/Sources/Settings/Storyboardable.swift [hunk 2]
```diff
@@ -0,0 +1,8 @@
+//
+//  File.swift
+//  
+//
+//  Created by Mikhail Rubanov on 11.08.2022.
+//
+
+import Foundation
```
Comment (drucelweisse) [inline VoiceOver Designer/Features/Sources/Settings/Storyboardable.swift:8]: Will it contain storyboardable or not implemented yet?

**Review by drucelweisse — APPROVED** [review]

**Comment by bbsody** [comment]
@akaDuality, I think it's OK to have it like that, and that's what Apple does in their macOS apps. 

When I first suggested showing all the controls in the disabled state, we hadn't considered adding accessibility containers support. AFAIK, accessibility containers have other options meaning that the right panel will have at least two appearances based on the current selection, so there is no point in showing disabled controls.

I want to refine the UI, though. Could you vertically centre the 'Select or draw an element to adjust its settings' label?

File: VoiceOver Designer/Features/Sources/Settings/Storyboardable.swift (hunk 2 above)
Comment (akaDuality) [reply VoiceOver Designer/Features/Sources/Settings/Storyboardable.swift]: Yeah, it should, but I haven't managed to implement it for SPM :-(

Comment (akaDuality) [reply VoiceOver Designer/Features/Sources/Settings/Storyboardable.swift]: Removed files

**Comment by akaDuality** [comment]
@bbsody Like that?

Screen Shot 2022-08-11 at 10 29 17

**Comment by akaDuality** [comment]
@bbsody Probably, containers wouldn't have separate settings, we will just group and give a name for the group directly in left panel.

But, `isModal` setting should be placed somewhere in the future...

**Comment by bbsody** [comment]
> @bbsody Like that?
> 
> 
> 
> Screen Shot 2022-08-11 at 10 29 17
> 
> 

Yes!

**Pull request merged**